acceptance/tests/test_package_version_parameter.rb

```diff
@@ -0,0 +1,42 @@
+require 'beaker-puppet'
+require_relative '../helpers'
+
+# Attempts to find the previous semver-based relase of the given agent version
+def some_prevous_version(version)
```

Comment:
Is this magic necessary? I think it'd simplify things if people were required to specify the `FROM` and `TO` environment variables.